jobs/ci-kubernetes-e2e-gce-1-6-master-upgrade-cluster.env

```diff
@@ -0,0 +1,12 @@
+### job-env
```

Comment:
Can we have a short comment explaining what kind of upgrade tests are performing here? For example, a lot of engineers are confused about what are the difference between this one and the below one with -master at the end.